_posts/2024-08-30-zarr-v3-specification.markdown

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "What's new in Zarr V3 Specification?"
+description: Blog Post on Zarr V3 Specification
+date: 2024-08-30
+categories: blog
+permalink: /zarr-v3/
+---
+
+## Hi, Zarr Community! 👋🏻
+
+I hope you're doing well! We recently released the first and second alpha
+versions of Zarr-Python V3; check [here](https://pypi.org/project/zarr/#history). 
+With the official release around the corner, there's a lot to look forward to.
+But before we dive headfirst into integrating Zarr-Python V3 into our workflows,
+I want to take a moment to provide an overview of the key changes and enhancements
+we've made in this new version.
```

Comment:
Clarify of the spec not the library.

Reply:
Fixed in https://github.com/zarr-developers/blog/pull/60/commits/7d3e9749df791acc9f51b9f30eab6a5baa612193.